policyengine_us/variables/gov/states/md/tax/income/agi/subtractions/md_two_income_subtraction.py

```diff
@@ -0,0 +1,61 @@
+from policyengine_us.model_api import *
+
+
+class md_two_income_subtraction(Variable):
```

Comment:
Could you please add a unit test for this?

Reply:
There is an integration test as part of this pull request, but I'll add a unit test when I return to the office on Friday, December 16.